bbb_broker.php

```diff
@@ -665,11 +666,17 @@ function bigbluebuttonbn_broker_recording_ready($params, $bigbluebuttonbn) {
     }
     // Sends the messages.
     try {
-        bigbluebuttonbn_send_notification_recording_ready($bigbluebuttonbn);
+        // That if was for trying to workaround the several emails sent for one record ready
+        if (bigbluebuttonbn_get_count_callback_event_log($decodedparameters->record_id) == 0) {
+            bigbluebuttonbn_send_notification_recording_ready($bigbluebuttonbn);
+        }
+        $overrides = array('meetingid' => $bigbluebuttonbn->meetingid);
```

Comment:
Why are you overriding meetingid with the same value that would be naturally assigned?

$log->meetingid = $bigbluebuttonbn->meetingid;